cmd/launcher/desktop.go

```diff
@@ -18,8 +17,11 @@ import (
 	"github.com/kolide/kit/ulid"
 	"github.com/kolide/launcher/ee/desktop/menu"
 	"github.com/kolide/launcher/ee/desktop/notify"
-	"github.com/kolide/launcher/ee/desktop/server"
+	rootserver "github.com/kolide/launcher/ee/desktop/runner/server"
+	desktopserver "github.com/kolide/launcher/ee/desktop/server"
```

Comment:
wonder if we should just go ahead an name these packages `rootserver` and `desktopserver`

Reply:
rootserver is the server run by the root process

desktopserver is the server run by the desktop process

maybe we should call them `rootprocserver` and `desktopprocserver`

any suggestions?